test/corpus/preprocessor.txt

```diff
@@ -153,36 +149,36 @@ class Of1879 {
 (compilation_unit
   (class_declaration
     name: (identifier)
-    body: (declaration_list
+    (declaration_list
       (method_declaration
-        type: (predefined_type)
+        returns: (predefined_type)
         name: (identifier)
         parameters: (parameter_list)
         body: (block
-          (line_directive
-            (preproc_integer_literal)
-            (preproc_string_literal))
-          (comment)
-          (line_directive)
-          (line_directive)
-          (line_directive
-            (preproc_integer_literal)
-            (preproc_integer_literal)
-            (preproc_integer_literal)
-            (preproc_integer_literal)
-            (preproc_integer_literal)
-            (preproc_string_literal))
-          (line_directive
-            (preproc_integer_literal)
-            (preproc_integer_literal)
-            (preproc_integer_literal)
-            (preproc_integer_literal)
-            (preproc_string_literal))
-          (line_directive
-            (preproc_integer_literal)
-            (preproc_string_literal))
-          (line_directive)
-          (line_directive))))))
+          (preproc_call
```

Comment:
This feels like a definite step backwards in terms of usefulness.

Reply:
do you mean the granular parsing of the line preprocessor directive? this is more how C handles it, but C# might be different because it has a fixed set of directives. I could add back the granularity, but I initially thought it's probably better to handle it like C